kernel/arch/dreamcast/hardware/biosfont.c

```diff
@@ -285,29 +290,31 @@ unsigned char bfont_draw_ex(uint8 *buffer, uint32 bufwidth, uint32 fg, uint32 bg
 }
 
 /* Draw half-width kana */
-unsigned char bfont_draw_thin(void *b, uint32 bufwidth, uint8 opaque, uint32 c, uint8 iskana) {
-    return bfont_draw_ex((uint8 *)b, bufwidth, bfont_fgcolor, bfont_bgcolor, (bfont_32bit ? (sizeof (uint32)) : (sizeof (uint16))) << 3, opaque, c, 0, iskana);
+size_t bfont_draw_thin(void *b, uint32_t bufwidth, bool opaque, uint32_t c, bool iskana) {
+    return bfont_draw_ex((uint8_t *)b, bufwidth, bfont_fgcolor, bfont_bgcolor, 
```

Comment:
```suggestion
    return bfont_draw_ex(b, bufwidth, bfont_fgcolor, bfont_bgcolor, 
```
Now that the underlying function is also void*, there's no need for this.